src/service/video-manager-impl.js

```diff
@@ -182,15 +182,13 @@ export class VideoManager {
    * @private
    */
   registerCommonActions_(video) {
-    // TODO(choumx, #9699): HIGH for unmuted play, LOW for muted play.
-    video.registerAction('play', video.play.bind(video, /* isAutoplay */ false),
-        ActionTrust.MEDIUM);
-    // TODO(choumx, #9699): LOW.
-    video.registerAction('pause', video.pause.bind(video), ActionTrust.MEDIUM);
-    video.registerAction('mute', video.mute.bind(video), ActionTrust.MEDIUM);
-    // TODO(choumx, #9699): HIGH.
-    video.registerAction('unmute', video.unmute.bind(video),
-        ActionTrust.MEDIUM);
+    // Only require ActionTrust.LOW for video actions to defer to platform
+    // specific handling (e.g. user gesture requirement for unmuted playback).
+    video.registerAction('play',
```

Comment:
Not at the moment since we don't have any LOW trust events. But since we are keeping `unmute` at LOW, if we ever expose a LOW trust event (e.g. `onload` for `amp-list`?) then on certain platforms unmuted playback can happen.